src/main/util/const.py

```diff
@@ -0,0 +1 @@
+RESOURCE_FOLDER_PATH = 'src/test/resources'
```

Comment:
1. Add copyright, please

2. I am not sure but maybe should you add a variable with root folder: `ROOT_DIR = os.path.dirname(os.path.abspath(__file__)) + '/../../` and use it `TEST_RESOURCES_PATH = os.path.join(ROOT_DIR, 'test/resources')`?
Rename, please, the variable, because you have test's and src's resources.

Reply:
I think that second proposal is nice idea!